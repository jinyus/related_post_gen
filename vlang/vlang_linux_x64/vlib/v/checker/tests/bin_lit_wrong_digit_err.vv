fn main() {
    println(0b1112)
}
