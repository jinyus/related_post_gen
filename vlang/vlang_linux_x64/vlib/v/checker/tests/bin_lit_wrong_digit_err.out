vlib/v/checker/tests/bin_lit_wrong_digit_err.vv:2:18: error: this binary number has unsuitable digit `2`
    1 | fn main() {
    2 |     println(0b1112)
      |                  ^
    3 | }
